.vib/mysql/vib-verify.json

```diff
@@ -17,13 +17,37 @@
       ]
     },
     "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/mysql"
+        },
+        "runtime_parameters": "YXV0aDogCiAgcm9vdFBhc3N3b3JkOiBDb21wbGljYXRlZFBhc3N3b3JkMTIzNAo=",
+        "target_platform": {
+          "target_platform_id": "{VIB_ENV_TARGET_PLATFORM}",
+          "size": {
+            "name": "M4"
```

Comment:
Have you tried using `S4`? As we are not performing Cypress tests this time, it may be enough

Reply:
Good catch! I´ve reduced the cluster size.